tests/tests/test_mtls.py

```diff
@@ -252,6 +253,10 @@ def test_mtls_enterprise(self, setup_ent_mtls, algorithm):
             )
             deploy.upload_image(artifact)
 
+        for device in devauth.get_devices_status("pending"):
+            devauth.decommission(device["id"])
+        time.sleep(self.wait_for_device_timeout_seconds)
```

Comment:
Is there any way we can test for a condition instead of waiting the full sleep every time? It is both time consuming and error prone. Typically we find that, one day when GCP is loaded, this sleep will not be long enough.

Reply:
roger that.
how about now?